mypyc/ir/rtypes.py

```diff
@@ -443,28 +443,88 @@ def deserialize(cls, data: JsonDict, ctx: 'DeserMaps') -> 'RTuple':
 )
 
 
+def compute_rtype_size(typ: RType) -> int:
```

Comment:
We need a separate function for calculating alignment, since alignment is different from the size for struct types.